package.json

```
@@ -5,7 +5,7 @@
   "repository": "https://github.com/Loskir/telegraf-media-group.git",
   "author": "Loskir",
   "license": "MIT",
-  "dependencies": {
-    "telegraf": "^3.36.0"
+  "peerDependencies": {
+    "telegraf": "^3.39.0"
```

```suggestion
    "telegraf": "^3.36.0 || ^4.0"
```

I think it shouldn't be the latest version, but rather the first supported one. I'm too lazy to check so let's leave it as is. Also added v4.

---

I added v4, but it looks like when installing modules for tests, I missed it and only v3 got into the commit, I should be more careful ....